<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Country</class>
 <widget class="QWidget" name="Country">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitar</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="countryCheckBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Marcar</string>
   </property>
  </widget>
  <widget class="QLabel" name="countryLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Peru</string>
   </property>
  </widget>
  <widget class="QLabel" name="capitalLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lima</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
